app/users/delete/template.hbs

```diff
@@ -5,6 +5,6 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    Are you sure you wish to delete the user {{model.name}}?
+    {{t 'messages.are_you_sure_delete'}}{{model.name}}?
```

Comment:
@cfleschhut  Thank you for the suggestion i didn't know that, i'll try to use this technique next time.